modules/appeals_api/spec/requests/v1/notice_of_disagreements_controller_spec.rb

```diff
@@ -91,26 +91,6 @@ def base_path(path)
         expect(nod.board_review_option).to eq('hearing')
       end
     end
-
-    context 'keeps track of which version of the api it is serving' do
```

Comment:
We still want this - just because the version may be going away doesn't mean we don't want to track which version is being used.

Reply:
We were doing this because 'v2' was rerouted to 'v1'. You still want this around?

Reply:
It's still a core part of the controller, yes. Every appeal record should store which api version was used to create it (and which pdf version we rendered it with when that job happens).